src/adapter/cloudflare-pages/handler.test.ts

```diff
@@ -49,3 +50,75 @@ describe('Adapter for Cloudflare Pages', () => {
     expect(() => handler({ request })).toThrowError('Custom Error')
   })
 })
+
+describe('Middleware adapter for Cloudflare Pages', () => {
+  it('Should return the middleware response', async () => {
+    const request = new Request('http://localhost/api/foo', {
+      headers: {
+        Cookie: 'my_cookie=1234',
+      },
+    })
+    const env = {
+      TOKEN: 'HONOISCOOL',
+    }
+    const handler = handleMiddleware(async (c, next) => {
+      const cookie = getCookie(c, 'my_cookie')
```

Comment:
Included this as an example of my use-case, but it's not entirely necessary as part of the test